### ISSUE TITLE
Doesn't work lilToon Gem

### ISSUE THREAD

**Issue by Kieaer** [lilxyzw/lilToon#24]
```
NullReferenceException: Object reference not set to an instance of an object
lilToon.lilMaterialProperty.get_floatValue () (at Assets/lilToon/Editor/lilMaterialProperty.cs:23)
lilToon.lilToonInspector.BlendOpFASetting () (at Assets/lilToon/Editor/lilInspector.cs:5752)
lilToon.lilToonInspector.DrawLightingSettings () (at Assets/lilToon/Editor/lilInspector.cs:5742)
lilToon.lilToonInspector.DrawAdvancedGUI (UnityEngine.Material material) (at Assets/lilToon/Editor/lilInspector.cs:1623)
lilToon.lilToonInspector.DrawAllGUI (UnityEditor.MaterialEditor materialEditor, UnityEditor.MaterialProperty[] props, UnityEngine.Material material) (at Assets/lilToon/Editor/lilInspector.cs:807)
lilToon.lilToonInspector.OnGUI (UnityEditor.MaterialEditor materialEditor, UnityEditor.MaterialProperty[] props) (at Assets/lilToon/Editor/lilInspector.cs:756)
UnityEditor.MaterialEditor.PropertiesGUI () (at <a259d3c004024353a2c217da97495055>:0)
UnityEditor.MaterialEditor.OnInspectorGUI () (at <a259d3c004024353a2c217da97495055>:0)
UnityEditor.UIElements.InspectorElement+<>c__DisplayClass58_0.<CreateIMGUIInspectorFromEditor>b__0 () (at <a259d3c004024353a2c217da97495055>:0)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)
```

version 1.3.1

## How to reproduce
1. Create new material
2. Set shader to lilToon
3. Select Rendering mode to Gem
4. Error occured. Inspector is empty and can't set anything.

**Comment by lilxyzw** [comment]
Thank you for the bug report. This error has been fixed in ad79060 and will be updated at a later date.

**Comment by lilxyzw** [comment]
A fix for this problem has been applied in lilToon 1.3.2 and later releases.
